2_pvc_destination_gen/tasks/adjust_pvs.yml

```diff
@@ -0,0 +1,21 @@
+- delegate_to: "{{ node_name }}"
+  block:
+  - name: "Finding PVCs associated with {{ node_name }}"
+    set_fact:
+      found_pvcs: "{{ pvc_data | json_query(query) | flatten }}"
+    vars:
+      query: "[*].pvcs[?node_name=='{{ node_name }}']"
+
+  - include_tasks: ./compute_adjustment.yml
+    loop: "{{ found_pvcs }}"
+    vars:
+      pvc_name: "{{ current_pvc.pvc_name }}"
+      bound_pod_uid: "{{ current_pvc.bound_pod_uid }}"
+      volume_name: "{{ current_pvc.volume_name }}"
+      capacity: "{{ current_pvc.capacity }}"
+      namespace: "{{ current_pvc.pvc_namespace }}"
+    loop_control:
+      loop_var: "current_pvc"
+    ignore_errors: true   # TODO: is this the right thing to do?
```

Comment:
On this one, what is the consequence if we fail mid-process here? Can the next step still run? Will it run with original PVC sizes, or will it error in an unexpected way?

Reply:
@djwhatle If we fail here, the loop will continue processing the next PVC.